### ISSUE TITLE
Same resources in 2 sections

### ISSUE THREAD

**Issue by yash251** [FrancescoXX/free-Web3-resources#166]
**2 sections have same resources**
Sections *Identity* and *Ethereum Development Tools* have same resources.

**To Reproduce**
1. Go to 'README.md'
3. Scroll down to 'Identity' section.
4. Remove the following
```
[Rainbow Dev Kit](https://www.rainbowkit.com/) RainbowKit is a React library that makes it easy to add wallet connection to your dapp.

[scaffoldETH](https://github.com/scaffold-eth/scaffold-eth) Scaffold-eth is everything you need to get started building decentralized applications on Ethereum!

[WAGMI](http://wagmi.sh/) wagmi.sh is a collection of React Hooks containing everything you need to start working with Ethereum.
```

**Comment by yash251** [comment]
@FrancescoXX please confirm

**Comment by krau5** [comment]
Hey, @yash251, can I be assigned to this issue later when it'll be confirmed?

**Comment by FrancescoXX** [comment]
correct @yash251 , they should be removed

**Comment by abhinow01** [comment]
hey @FrancescoXX , i am new to contributing to opensource , i would love to work on this issue , can this be assigned to me ?

**Comment by yash251** [comment]
Also the youtube channels section is repeated

**Comment by yash251** [comment]
@FrancescoXX fixed. please check

**Comment by FrancescoXX** [comment]
fixed by #177
